Tests/Common/Util/LeanDataPathComponentsTests.cs

```diff
@@ -48,7 +49,7 @@ public TestCaseData[] GetTestCases()
                 from resolution in resolutions
                 from tickType in tickTypes
                 let date = resolution == Resolution.Hour || resolution == Resolution.Daily ? DateTime.MinValue : referenceDate
-                let name = string.Format("{0}-{1}-{2}-{3}", securityType.ToLower(), market.ToLower(), resolution.ToLower(), tickType.ToLower())
+                let name = $"{securityType}-{market}-{resolution}-{tickType}".ToLowerInvariant()
```

Comment:
Missing the individual `.ToStringInvariant`?

Reply:
Updated to use `using static` -- `Invariant(fs)` pattern